### PR TITLE
[5.4] Add Event timezone in inTimeInterval $now

### PR DESCRIPTION
**Pull request by pavinthan**
It's will reduce timezone issue in scheduling, please have a look into this.

### CODE REVIEW

**Comment by tillkruss** [comment]
Can you add a test for this, please?

**Comment by pavinthan** [comment]
I've updated the test, please have a look into this, Thanks.

File: src/Illuminate/Console/Scheduling/ManagesFrequencies.php [hunk 1]
```diff
@@ -53,9 +53,11 @@ public function unlessBetween($startTime, $endTime)
     private function inTimeInterval($startTime, $endTime)
     {
         return function () use ($startTime, $endTime) {
-            $now = Carbon::now()->getTimestamp();
+            $now = Carbon::now($this->timezone)->getTimestamp();
```
Comment (lucasmichot) [inline src/Illuminate/Console/Scheduling/ManagesFrequencies.php]: Why not just :
```php
return Carbon::now($this->timezone)->between(
    Carbon::parse($startTime, $this->timezone),
    Carbon::parse($endTime, $this->timezone),
    true
);
```

?

**Comment by pavinthan** [comment]
@lucasmichot 👍

**Comment by taylorotwell** [comment]
This changes code unrelated to the timezone. Only make the change specified in the title.

**Comment by themsaid** [comment]
This fixes the following case:

```
$schedule->command('inspire')
    ->everyMinute()
    ->timezone('Africa/Cairo')
    ->between('2017-07-10 14:25', '2017-07-10 15:25');
```

At `2017-07-10 14:25` the UTC time is `2017-07-10 12:25`, without this PR we currently compare the time without respect to the specified timezone, so this command will wait until UTC time is `14:25` which is two hours after it should run.

I believe this fix is ok.

**Comment by pavinthan** [comment]
Thank you guys 👍

**Pull request merged**